dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -939,6 +939,24 @@ def test_min_col_samples_used(self, *mocks):
                                                samples_per_update=1)
         self.assertEqual(2, sparse_profile._min_col_samples_used)
 
+    @mock.patch('dataprofiler.profilers.profile_builder.DataLabeler')
+    def test_min_true_samples(self, *mocks):
+        empty_df = pd.DataFrame([])
+
+        # Test invalid input
+        msg = "`min_true_samples` must be an integer or `None`."
+        with self.assertRaisesRegex(ValueError, msg):
+            profile = dp.StructuredProfiler(empty_df, 
+                                            min_true_samples="Bloop")
+
+        # Test None input (defaults to zero)
```

Comment:
You say "defaults to zero" but then assert that its None. I would clarify in the comment that it doesn't get set to 0 in the init, otherwise the comment seems to clash with the assertion being made

Reply:
Fair, changed the wording to make it somewhat more clear